src/components/UploadSection.svelte

```diff
@@ -14,6 +23,41 @@
         }
     });
 
+    async function sendFiles() {
+        for (let i: number = 0; i < files.length; i++) {
+            let exifData = null;
+
+            try {
+                let exifData = await exifr.parse(files[i]);
+            } catch (e) {
+                console.error(e);
+            }
+
+            let takenAtSec = null;
+
+            if (exifData) {
+                takenAtSec = exifData.DateTimeOriginal.valueOf();
+            }
+
+            let postFile: PostRequest = {
+                ObjectKey: files[i].name,
+                SizeBytes: files[i].size,
+                TakenAtSec: takenAtSec,
+            };
+
+            postRequest.push(postFile);
+        }
+        try {
+            postResponse = await postInsertRequest(postRequest);
```

Comment:
Good naming. I like this, just want to continue your idea a little bit further.
In future we will have more POST requests for other stuff (e.g. restore requests or user creation). 

So we need a proper naming. 

Usually, when working with requests, responses we do the next namings:

- func name: `createUploadRequests`
- parameter name: `CreateUploadRequestRequest` (... well we have a confusion here, because domain object name is `UploadRequest`
- response name: `CreateUploadRequestResponse`

So the pattern is <verb><domain object>(request or response)

Maybe we should rename the domain object :thinking:

Reply:
Maybe, but think things woth "CreateUploadRequestRequest" or "CreateUploadRequestResponse" not so bad, but just in this case do we can for request leave inly one request dies it affects on undestanding?